### DISCUSSION TITLE
Welcome to pytkfaicons Discussions!

### DISCUSSION THREAD

**Discussion by kr-g** [kr-g/pytkfaicons#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by avalon60** [comment]
Hi is there a sample  code snippet, showing how to use the library?

**Reply by kr-g** [reply]
Hello Clive.
Please have a look at

https://github.com/kr-g/pytkfaicons/blob/main/run_sample.py
